src/main/java/Zoo.java

```diff
@@ -0,0 +1,47 @@
+import animals.Animal;
+import animals.Penguin;
+import animals.Shark;
+import animals.Sparrow;
+import animals.Tiger;
+import areas.Aviary;
+import areas.BirdAviary;
+import areas.Oceanarium;
+import interfaces.Flyable;
+import interfaces.Swimmable;
+import java.util.ArrayList;
+
+public class Zoo {
+    public static void main(String[] args) {
```

Comment:
let's move main method to separate class, it's better to separate logic ( storing animals and starting program, initializing zoo is different purposes)

Reply:
done